indexer_util/indexer_util/indexer_util.py

```diff
@@ -88,20 +88,51 @@ def _wait_elasticsearch_healthy(es):
     logging.getLogger("elasticsearch").setLevel(logging.INFO)
 
 
-def maybe_create_elasticsearch_index(elasticsearch_url, index_name):
-    """Creates Elasticsearchindex if it doesn't already exist."""
-    es = Elasticsearch([elasticsearch_url])
+def get_es_client(elasticsearch_url):
+    es = Elasticsearch([elasticsearch_url],
+                       retry_on_timeout=True,
```

Comment:
```
# Retry flags needed for large datasets.
```
Otherwise, someone might delete the flags, see that 1000 Genomes indexes fine, and think the flags aren't needed.

Reply:
Done.